client/src/components/v2/CameraRecording.js

```diff
@@ -0,0 +1,50 @@
+import React from 'react';
+import { Card } from 'react-bootstrap';
+import Button from 'react-bootstrap/Button';
+import PropTypes from 'prop-types';
+import { capitalize } from 'lodash';
+import { startRecording, stopRecording } from 'api/v2/camera';
+import CameraRecordingStatus from './CameraRecordingStatus';
+import styles from './CameraRecording.module.css';
+
+/**
+ * @typedef {object} CameraRecordingProps
+ * @property {Array} devices Array of strings of device names
+ */
+
+/**
+ * Camera recording buttons for starting and stopping video recording.
+ *
+ * Starts/stops recording for both displays.
+ *
+ * This is a feature intended for V3 but is currently in V2 for testing.
+ *
+ * @param {CameraRecordingProps} props Props
+ * @returns {React.Component<CameraRecordingProps>} Component
+ */
+export default function CameraRecording({ devices }) {
+  return (
+    <Card>
+      <Card.Body>
+        <Card.Title>Recording Controls</Card.Title>
+        {
+          devices.map((device) => (
+            <div styles={styles.device} key={device}>
+              <Card.Subtitle>{capitalize(device)}</Card.Subtitle>
```

Comment:
I'm wary of importing from lodash and other external libraries where possible to reduce bloat and bundle size. This isn't the only instance where we want to derive a pretty name from the camera display name. See [this](https://github.com/monash-human-power/dashboard/blob/e92c459fe32fb620c96bcd2d53a90f7f971b0ae0/client/src/components/v2/CameraSettings.js#L21). Now that it's more useful it would be handy to refactor this in to a "getPrettyDeviceName" API function.

Reply:
Done
da1d043

Reply:
Actually done now
4920ea2